OracleFMWInfrastructure/dockerfiles/12.2.1.3/properties/domain.properties

```diff
@@ -0,0 +1,5 @@
+username=mywlsusername
+password=mywlspassword
+db_user=sys
+db_pass=Oradoc_db1
+db_schema=Oradoc_db1
```

Comment:
hmm, there are normally a number of schemas.... usually we use a PREFIX and then add on to the end what is needed... not sure how many INFRA alone creates, have to check in RCU

Reply:
these are properties only for the db in a container.  The RCU properties like the PREFIX are under properties/setEnv.txt